configure should detect Python version and allow builder to override
detection via --with-python or a PYTHON environment variable, and then the
build process should replace the first line of the script instead.

Need to determine if we need to file an upstream bug.

--- gtk+-2.24.28/gtk/gtk-builder-convert	Thu May  7 22:49:42 2015
+++ gtk+-2.24.28/gtk/gtk-builder-convert	Thu May  7 22:50:51 2015
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.7
 #
 # Copyright (C) 2006-2008 Async Open Source
 #                         Henrique Romano <henrique@async.com.br>
